as.c

```diff
@@ -417,13 +446,27 @@ DownloadProfile(HANDLE hWnd, const WCHAR *host, const char *username, const char
         goto done;
     }
 
+    /* check content-type if specified */
+    if (strlen(comps->content_type) > 0)
+    {
+        char tmp[256];
+        DWORD len = sizeof(tmp);
+        BOOL res = HttpQueryInfoA(hRequest, HTTP_QUERY_CONTENT_TYPE, tmp, &len, NULL);
```

Comment:
Any reasons not to use W variant?

Reply:
Just not to give the impression that we support arbitrary unicode characters in this header string. I think MIME-types have to be us-ascii.